src/app/organizations/organization/organization.component.html

```diff
@@ -28,8 +28,9 @@
     </mat-card>
     <mat-card fxFlex class="my-3">
       <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="10px">
-        <div>
-          <img src="https://via.placeholder.com/150" alt="Logo" class="img-circle">
+        <div >
+          <img *ngIf="org?.avatarUrl == null" src="https://via.placeholder.com/150" class="img-circle" height="150" width="150">
```

Comment:
Copy this into our repo, just in case.

Reply:
Added new image to repo I found doing creative commons search.